src/components/SpaceItem.vue

```diff
@@ -0,0 +1,33 @@
+<script lang="ts" setup>
+import type { Space as SpaceType } from '@/types';
```

Comment:
I started dropping `import type` convention for pure type imports recently, because it started feeling cumbersome, with not much of benefit. Unless you have preference for those kinds of imports, we should start dropping those from new code.